src/MultiSelect/MultiSelect.tsx

```diff
@@ -220,33 +233,30 @@ const MultiSelect: React.FC<Props> = ({
         {...getMenuProps()}
       >
         {isOpen &&
-          (selectableOptions.length > 0 ? (
-            selectableOptions.map((o, i) => (
-              <li
-                className={classNames(
-                  cssClass.MENU_OPTION,
-                  i === highlightedIndex && cssClass.MENU_OPTION_HIGHLIGHTED,
-                )}
-                key={`${o.value}${i}`}
-                {...getItemProps({ item: o, index: i })}
-              >
-                {o.content || o.value}
-              </li>
-            ))
+          // slice out the dummy "add item X" option
+          (selectableOptions.slice(1).length === 0 && (!creatable || hasExactMatch) ? (
```

Comment:
```suggestion
          (selectableOptions.length === 1 && (!creatable || hasExactMatch) ? (
```
Since the sliced array isn't actually being used, this way would be a little simpler and saves the computer some work.

Reply:
There's a change that the selectableOptions will *correctly* be 1 item, which is if the typeahead matches exactly one item

Reply:
Ohhh I see, so the equivalent length check would actually be `selectableOptions.length <= 1`

Reply:
oh maybe I misunderstood you. Actually `=== 1` might work here. I can take a look